<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutBorneFontaineUI</class>
 <widget class="QDialog" name="AjoutBorneFontaineUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjoutBorneFontaineUI</string>
  </property>
  <widget class="QLabel" name="label_salaire">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitude :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nomTopographique_stat">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ville_stat">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_codeDepartement">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrondissement :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_commission">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Latitude :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dateNaissance">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom topographique :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titre">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AJout dâ€™une borne fontaine</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prenom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant de la voie publique:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_idVoiePublique_stat">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_quantite">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="identifiant_statBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_latitude_stat">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>14</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_longitude_stat">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>14</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_arrondissement_stat">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AjoutBorneFontaineUI</receiver>
   <slot>validerBorneFontaine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerBorneFontaine()</slot>
 </slots>
</ui>
